Update notifications.yml

maxwellfunk opened the pull request:
@lachellel Please find the associated announcement for the DigiCert SSP CAs being established for Dept of Ed.  Assumed the incremental name of the new issuing CA based on historical naming convention.

maxwellfunk committed:
Update notifications.yml

Assumed the incremental name of the new issuing CA based on historical naming convention.

lachellel commented:
@maxwellfunk let's wait just a few more hours until we have confirmation on those CA names and the CDP and OCSP URIs.  Keeping open for edits.

maxwellfunk committed:
Update notifications.yml

Updated to reflect that DigiCert only only going to issue a Device CA (agency CA reference removed.
